### ISSUE TITLE
warning: method redefined; discarding old find_spec_for_exe

### ISSUE THREAD

**Issue by Aqualon** [bundler/bundler#4909]
If I execute the following line
```
RUBYOPT="-W2" bundle exec ruby -e "p 1"
```
I get those two warnings:
```
/Users/bruno.escherl/.rvm/gems/ruby-2.3.1/gems/bundler-1.12.5/lib/bundler/rubygems_integration.rb:468: warning: method redefined; discarding old find_spec_for_exe
/Users/bruno.escherl/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems.rb:261: warning: previous definition of find_spec_for_exe was here
```
Is this something that should be fixed?

**Comment by segiddins** [comment]
This should be addressed in the 1.13 release candidates

**Comment by Aqualon** [comment]
Thanks, tried `bundler-1.13.0.rc.2` and the warning is gone.
